foundation/org.eclipse.persistence.core/src/org/eclipse/persistence/expressions/ExpressionOperator.java

```diff
@@ -2213,6 +2217,14 @@ public void printCollection(Vector items, ExpressionSQLPrinter printer) {
                 printer.printString(alias.getNameDelimited(printer.getPlatform()));
             } else if ((this.selector == Count) && (item.isExpressionBuilder())) {
                 printer.printString("*");
+            } else if ((this.selector == Case) && (item.isConstantExpression())) {
```

Comment:
Expression.java line 4646:

   /**
     * ADVANCED:
     * Return an expression on the literal.
     * A literal is a specific SQL syntax string that will be printed as is without quotes in the SQL.
     * It can be useful for printing database key words or global variables.
     * <p>Example:
     * <blockquote><pre>
     * reportQuery.addItem("currentTime", builder.literal("SYSDATE"));
     * </pre></blockquote>
     */
    public Expression literal(String literal) {
        return new LiteralExpression(literal, this);
    }

I don't think we want Literals here.

Reply:
I agree, we don't want to use Literals here